src/components/Footer/index.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <footer
+      className="flex text-white text-center py-4 mt-auto h-100 md:h-40 sm:h-auto items-center justify-center"
+      data-testid="footer"
+      style={{ backgroundColor: "#2F2E2E" }}
```

Comment:
I suggest moving the inline styles to the className since Tailwind can handle the background color. You can simply add the `bg-[#2F2E2E]` class in the className instead of using style={{}}